src/cloudant/document.py

```diff
@@ -351,5 +401,5 @@ def put_attachment(self, attachment, content_type, data, headers=None):
             headers=headers
         )
         resp.raise_for_status()
-
+        self.fetch()
```

Comment:
Is it not possible to `add`|`remove` the attachment from the local dictionary if the `put`|`delete` request was successful in place of re-fetching the whole document?